docs/source/getting-started.md

```diff
@@ -0,0 +1,194 @@
+# Getting Started with ExecuTorch
+This section is intended to describe the necessary steps to take PyTorch model anda run it using ExecuTorch. To use the framework, you will need to take the following steps:
+- Install the ExecuTorch python package and runtime libraries.
+- Export the PyTorch model for the target hardware configuration.
+- Run the model using the ExecuTorch runtime APIs.
```

Comment:
Let's decouple running on host vs deploying to target platform. I'd suggest this:

- Run the model using the ExecuTorch runtime APIs on your development platform.
- Deploy the model along with ExecuTorch runtime onto target platform.